flink/json-jobmanager.yml

```diff
@@ -0,0 +1,224 @@
+type: json
+data:
+  url: /v1/jobmanager/metrics?get=numRunningJobs,numRegisteredTaskManagers,taskSlotsAvailable,taskSlotsTotal,Status.JVM.Memory.Heap.Max,Status.JVM.Memory.Heap.Used,Status.JVM.Memory.Heap.Committed,Status.JVM.Memory.NonHeap.Max,Status.JVM.Memory.NonHeap.Used,Status.JVM.Memory.NonHeap.Committed,Status.JVM.Memory.Direct.Count,Status.JVM.Memory.Direct.MemoryUsed,Status.JVM.Memory.Direct.TotalCapacity,Status.JVM.Memory.Mapped.Count,Status.JVM.Memory.Mapped.MemoryUsed,Status.JVM.Memory.Mapped.TotalCapacity,Status.JVM.Threads.Count,Status.JVM.CPU.Time,Status.JVM.CPU.Load,Status.JVM.ClassLoader.ClassesLoaded,Status.JVM.ClassLoader.ClassesUnloaded
+  server: ${FLINK_HOSTPORT}
+
+observation:
+  - name: jobmanager_registered_taskmanagers_num
+    metricNamespace: flink
+    path: $.[?(@.id=numRegisteredTaskManagers)]
+    metric:
+      - name: jobmanager.registered.taskmanagers.num
+        source: value
+        type: long_gauge
+        label: num task managers
+        description: The number of registered taskmanagers
```

Comment:
taskmanagers -> should we need a space between task and manager? I see a space in the label.

Reply:
Fixed